webcompat/templates/user-activity.html

```diff
@@ -2,20 +2,20 @@
 {% block title %}{{user}}'s Activity{% endblock %}
 {% block body %}
 {% include "shared/nav.html" %}
-<main role="main" data-repo-path="{{config['ISSUES_REPO_URI']}}">
-  <div class="wc-Loader js-loader"></div>
-  <section class="wc-UISection">
-    <div class="wc-UIContent">
-      <h1 class="wc-Title wc-Title--l wc-ResetOffsetBottom js-username">
-        {{user}}'s activity
-      </h1>
-    </div>
+<main role="main" class="body-webcompat" data-repo-path="{{config['ISSUES_REPO_URI']}}">
+  <!-- <div class="wc-Loader js-loader"></div> -->
```

Comment:
again: just noting that we're commenting out the loader. whatever follow up issue we have to track that work should probably link to this line so we don't forget.

Reply:
note: the plan is to comment these out so we don't forget where they go later.